src/HtmlGenerator/Pass2-Finalization/SolutionFinalizer.cs

```diff
@@ -264,6 +311,7 @@ private void DeployFilesToRoot(string destinationFolder)
             FileUtilities.CopyDirectory(sourcePath, destinationFolder);
 
             StampOverviewHtmlWithDate(destinationFolder);
+            if (emitAssemblyList) ToggleSolutionExplorerOff(destinationFolder);

```

Comment:
I like surrounding single statements in a block but don't worry about it right now.